airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/avro/AvroConstants.java

```diff
@@ -4,10 +4,12 @@
 
 package io.airbyte.integrations.destination.s3.avro;
 
-public class S3AvroConstants {
+public class AvroConstants {
 
   // Field name with special character
   public static final String DOC_KEY_VALUE_DELIMITER = ":";
   public static final String DOC_KEY_ORIGINAL_NAME = "_airbyte_original_name";
+  // This name must match ab_additional_col in source_s3/source_files_abstract/stream.py
+  public static final String ADDITIONAL_PROPERTIES_FIELD_NAME = "_ab_additional_properties";
```

Comment:
Done. The additional properties field is renamed to `_airbyte_additional_properties` on the Avro side. On the Json side, it can either be `_airbyte_additional_properties` or `_ab_additional_properties`.